src/clr-angular/accordion/accordion-panel.html

```diff
@@ -1,5 +1,5 @@
 <ng-container *ngIf="panel | async; let panel">
-  <div *ngIf="panel.status !== AccordionStatus.Inactive" aria-live="assertive" class="clr-sr-only">
+  <div *ngIf="panel.status !== AccordionStatus.Inactive" [id]="'clr-accordion-header-status-' + panel.templateId" aria-live="assertive" class="clr-sr-only">
```

Comment:
this already has the approvals so I won't ding it. 😊 

but usually when i find myself doing things like concatenating the same string in multiple places, i'll make it a function or a method so i don't, say, change `clr-accordion-header-status` to something different in one place and forget about it in the other...

Reply:
Fixed up